includes/REST_API/Hotlinking_Controller.php

```diff
@@ -276,9 +288,9 @@ public function proxy_url( $request ) {
 			// Other than php://memory, php://temp will use a temporary file once the amount of data stored hits a predefined limit (the default is 2 MB).
 			// The location of this temporary file is determined in the same way as the {@see sys_get_temp_dir()} function.
 			if ( WP_DEBUG ) {
-				$stream_handle = fopen( 'php://temp', 'wb' ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_read_fopen
+				$stream_handle = fopen( 'php://memory', 'wb' ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_read_fopen
```

Comment:
What is the benefit of using memory of temp here?

Reply:
The comment above explains the difference well.

`php://temp` will use a temporary file once the amount of data stored hits a predefined limit  (the default is 2 MB), so touching the disk. `php://memory` is all in-memory.

Reply:
Yes, I read the 2 line comment. I wanted more details than that.